source/community/release_notes/sumac/dev_op_release_notes.rst

```diff
@@ -40,13 +102,28 @@ Other Operator Changes
 Deprecations & Removals
 ***********************
 
+- The existing "Content Libraries" feature has been renamed to "Legacy Libraries" and will be deprecated in the next release (Teak), and removed entirely in Ulmo in favor of the new "Content Libraries" feature. The Teak release will include a tool for migrating content from Legacy Libraries into new Content Libraries.
+- In `frontend-app-learner-dashboard <https://github.com/openedx/frontend-app-learner-dashboard>`_ 
+   - support for Optimizely has been removed along with the ProductRecommendations widget.
+   - `DEPR: Optimizely Support <https://github.com/openedx/frontend-app-learner-dashboard/issues/387>`_
+   - Removed the RecommendationsPanel widget
+   - `DEPR: RecommendationsPanel <https://github.com/openedx/frontend-app-learner-dashboard/issues/410>`_
+- In edx-platform:
+   - ``commerce-coordinator`` related code has been removed
+   - `PR: <https://github.com/openedx/edx-platform/pull/35527>`_
+- [UPCOMING] In Teak pre-design-tokens brand packages will no longer be supported. With design tokens, theme authors will instead override core Paragon tokens by defining their own JSON tokens that get deep merged alongside the core Paragon tokens, thus overriding any tokens that were defined by the theme author. See `the associated DEPR ticket for details <https://github.com/openedx/brand-openedx/issues/23>`_.
+- The cs_comments_service application is being replaced by forum. `[DEPR]: Replace cs_comments_service #437 <https://github.com/openedx/cs_comments_service/issues/437>`_
+- The Zooming Image Tool will be deprecated in Sumac. `[DEPR] Zooming Image Tool (HTML block template) #31436 <https://github.com/openedx/edx-platform/issues/31436>`_
```

Comment:
Another deprecation: 

The EdxRestApiClient` has been deprecated and removed in this release. See `[DEPR]: Complete removal of EdxRestApiClient` <https://github.com/openedx/public-engineering/issues/189>_ for details.

Reply:
more deprecations

```
The Demographics app has been removed. It was added to support a private edX Demographics IDA for collecting additional user info. See `[DEPR]: Demographics Django app #35127 <https://github.com/openedx/edx-platform/issues/35127>`_.

Reply:
Ok, one comment for the rest:

```
- The programs_listing endpoint in the credentials application has been removed. `[DEPR]: /program-listing endpoint <https://github.com/openedx/credentials/issues/2642>`_.
- The skill_level endpoint has been removed `[DEPR]: endpoint /user/v1/skill_level/{job_id}/ <https://github.com/openedx/edx-platform/issues/35302>`_.
- [UPCOMING] The Toggle 'block_structure.storage_backing_for_cache' will be removed, with a default setting of True. You may want to test enabling in Sumac before it becomes the default behavior in Teak. See `[DEPR]: block_structure.storage_backing_for_cache in edx-platform <https://github.com/openedx/public-engineering/issues/32>`_.
- 
```

This is everything from the most recent [DEPR WG meeting](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3996745737/DEPR+Meeting+Notes+2024). There may be additional new items in [Next Release: Sumac - Operator/Dev Notes ](https://openedx.atlassian.net/wiki/x/AQCw-g)